documentation/src/container_types/operators_hamiltonians_and_systems.md

```diff
@@ -4,10 +4,10 @@ Operators and systems act on a state space using HashMaps (Dictionaries) of oper
 
 For spins, the operators and systems represent
 \\[ 
-\hat{O} = \sum_{j=0}^N \alpha_j \prod_{k} \sigma^{k}_j \\\\
-    \sigma^{k}_j \in \\{ X, Y, Z, I \\}
+\hat{O} = \sum_{j} \alpha_j \prod_{k=0}^N \sigma_{j, k} \\\\
+    \sigma_{j, k} \in \\{ X_k, Y_k, Z_k, I \\}
 \\]
-where the \\(\sigma^{k}_j\\) are `SinglePauliOperators`.
+where the \\(\sigma_{j, k}\\) are `SinglePauliOperators`.
```

Comment:
Any particular reason for putting both indices down on the sigmas? 
Also, I would add the `k` subscript also on the identity: `I_k`

Reply:
Yes, the equation itself has been modified with Nico and we felt this was best